docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/310_press_archive.content.yml

```diff
@@ -39,12 +39,12 @@
       field_raw_html:
         - format: "raw_html"
           value: |
-            <p><a href="/news-events/press-releases/2025">2025</a>  |&nbsp;
+            <div class="usa-section"><p><a href="/news-events/press-releases/2025">2025</a>  |&nbsp;
```

Comment:
This should be `<div class="usa-section" data-eddl-landing-row="">` so that the analytics for the mini landing count it. 

`data-eddl-landing` is needed for each section of a landing page.